brickpi3/brickpi3_spi.c

```diff
@@ -24,7 +24,8 @@
 
 #include "brickpi3.h"
 
-#define BRICKPI3_MIN_FIRMWARE_VERSION	1002000 /* 1.2.0 */
+//define BRICKPI3_MIN_FIRMWARE_VERSION	1002000 /* 1.2.0 */
```

Comment:
Delete this line. We can always look at the git history if we need to know what it was in the past.